ozaria/site/components/teacher-dashboard/modals/ModalEditClass.vue

```diff
@@ -81,6 +82,9 @@ export default Vue.extend({
       getSessionsMapForClassroom: 'levelSessions/getSessionsMapForClassroom',
       courses: 'courses/sorted'
     }),
+    moreOptionsText () {
+      return this.moreOptions ? 'Less Options' : 'More Options'
```

Comment:
It should be `Hide Options`

Also, i18n the text.

If it's not too much effort, can we add a caret up and down symbol next to text?